horizon_steps: 2
time_step: 0.01
Q: [5.0, 0.0, 0.0, 0.0, 
    0.0, 5.0, 0.0, 0.0, 
    0.0, 0.0, 1.0, 0.0, 
    0.0, 0.0, 0.0, 1.0]
R: [1.0, 0.0, 0.0, 1.0]

rollout_number: 1
lambda: 1
vel_stdev: 1.0
ang_stdev: 0.1

bike_length: 0.48
